simple-chat-front/src/components/Chat/Chat.jsx

```diff
@@ -1,54 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
 import CreateRoom from '../Room/CreateRoom';
 import Room from '../Room/Room';
+import Message from '../Message/Message';
+import { BiMailSend, BiExit } from "react-icons/bi";
 import './Chat-Style.css'
-
-let roomsMockData = [
-    {
-        "RoomId": "1",
-    },
-    {
-        "RoomId": "2",
-    },
-    {
-        "RoomId": "3",
-    },
-    {
-        "RoomId": "4",
-    },
-    {
-        "RoomId": "5",
-    },
-    {
-        "RoomId": "6",
-    },
-    {
-        "RoomId": "7",
-    },
-    {
-        "RoomId": "8",
-    },
-]
+import roomsMockData from "../../utils/roomsMockData.json"
```

Comment:
for better conventions, test/mock data should be in a directory called `samples`